src/main/java/fi/aalto/cs/apluscourses/ui/base/ComponentUtil.java

```diff
@@ -0,0 +1,38 @@
+package fi.aalto.cs.apluscourses.ui.base;
+
+import java.awt.Font;
+import java.awt.font.TextAttribute;
+import java.util.Collections;
+import java.util.Map;
+import javax.swing.JComponent;
+
+/**
+ * A helper class for {@link JComponent} objects.
+ */
+public class ComponentUtil {
```

Comment:
This class could maybe go one one directory "upwards" and reside in `ui`? (I could also be misunderstanding how `base` is used here)

Reply:
A good point. With "base", I just meant non-domain-specific base classes that domain-specific classes can inherit, so yeah, this should be in a different package.